jekyll/_cci2/language-java-maven.md

```diff
@@ -96,7 +96,7 @@ jobs:
 
   build:
     docker:
-      - image: circleci/openjdk:stretch
+      - image: cimg/openjdk.17.0.1
```

Comment:
Typo

`cimg/openjdk.17.0.1` should be `cimg/openjdk:17.0.1`

Reply:
Fixed